Code/Framework/AzToolsFramework/AzToolsFramework/Entity/EditorEntityHelpers.cpp

```diff
@@ -468,6 +469,17 @@ namespace AzToolsFramework
         EntityIdList children;
         EditorEntityInfoRequestBus::EventResult(children, parentId, &EditorEntityInfoRequestBus::Events::GetChildren);
 
+        // If Prefabs are enabled, don't check the order for an invalid parent, just return its children (i.e. the root container entity)
```

Comment:
nit: Can you also add a line as to why this is okay with prefabs?

Reply:
Sure thing, done